internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -273,6 +314,22 @@ func (r *AgentServiceConfigReconciler) SetupWithManager(mgr ctrl.Manager) error
 		Complete(r)
 }
 
+func (r *AgentServiceConfigReconciler) deploymentStatus(ctx context.Context, log logrus.FieldLogger, deploymentName string, conditionType string) error {
+	deployment := &appsv1.Deployment{}
+	if err := r.Get(ctx, types.NamespacedName{Name: deploymentName, Namespace: "assisted-installer"}, deployment); err != nil {
+		return err
+	}
+	condiitons := &deployment.Status.Conditions
```

Comment:
Typo should be `conditions`

Reply:
Also, why have a separate pointer variable here? Couldn't the loop be:
`for _, condition := range deployment.Status.Conditions`?